src/main/java/github/weichware10/toolbox/gui/Tutorial.java

```diff
@@ -0,0 +1,104 @@
+package github.weichware10.toolbox.gui;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+import github.weichware10.toolbox.codecharts.CodeCharts;
+import github.weichware10.toolbox.zoommaps.ZoomMaps;
+import github.weichware10.util.Files;
+import github.weichware10.util.Logger;
+import github.weichware10.util.config.ConfigClient;
+import github.weichware10.util.db.DataBaseClient;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.text.Text;
+import javafx.stage.Stage;
+
+public class Tutorial {
+    private final Stage primaryStage;
+    private final ConfigClient configClient;
+    private final DataBaseClient dataBaseClient;
+    private TutorialController controller;
+    private static ArrayList<Image> imageList = new ArrayList<>();
+    private final String toolType;
+    
+    public Tutorial(Stage primaryStage, String toolType,
+                    ConfigClient configClient, DataBaseClient dataBaseClient) {
+        this.primaryStage = primaryStage;
+        this.toolType = toolType;
+        this.configClient = configClient;
+        this.dataBaseClient = dataBaseClient;
+
+        primaryStage.setTitle("Tutorial");
+
+        FXMLLoader loader = new FXMLLoader(getClass().getResource("Tutorial.fxml"));
+
+        Parent root = null;
+        try {
+            root = loader.load();
+        } catch (Exception e) {
+            Logger.error("Error when loading main scene", e, true);
+            System.exit(-1);
+        }
+
+        controller = loader.getController();
+        controller.setTutorial(this);
+
+        //Kümmert sich um das Laden aller Bilder und setzt das erste Bild
+        boolean finished = false;
+        int index = 0;
+        while (!finished) {
+            try {
+                String imageLoc = Files.saveImage(String.format("https://raw.githubusercontent.com/weichware10/dokumente/main/tutorial/%s/%d.png", toolType, index));
+                imageList.add(new Image(Paths.get(imageLoc).toUri().toString()));
+            } catch (IllegalArgumentException | IOException e) {
+                finished = true;
+            }
+        index += 1;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'while' child has incorrect indentation level 8, expected level should be 12.